hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AConfiguration.java

```diff
@@ -617,4 +624,135 @@ public void testSecurityCredentialPropagationEndToEnd() throws Exception {
         "override,base");
   }
 
+  @Test(timeout = 10_000L)
```

Comment:
There's already a test timeout ruleeout so you don't need `timeout = 10_000`.

Reply:
Moved to a separate unit test which doesn't need to extend any other class - so retaining the timeout.